tests/bsim/bluetooth/host/adv/long_ad/src/scanner.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdbool.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/logging/log.h>
+
+#include "ad.h"
+
+#include "babblekit/flags.h"
+#include "babblekit/testcase.h"
+
+LOG_MODULE_REGISTER(scanner, LOG_LEVEL_ERR);
+
+DEFINE_FLAG(scan_received);
+
+static bool data_parse_cb(struct bt_data *data, void *user_data)
+{
+	if (data->type == BT_DATA_NAME_COMPLETE) {
+		LOG_DBG("Complete Name:");
+		LOG_DBG("%s", data->data);
+	} else if (data->type == BT_DATA_NAME_SHORTENED) {
+		LOG_DBG("Short Name");
+		LOG_DBG("%s", data->data);
+	} else {
+		LOG_DBG("Type: %02x (size: %d)", data->type, data->data_len);
+		LOG_HEXDUMP_DBG(data->data, data->data_len, "Data:");
+	}
+
+	return true;
+}
```

Comment:
What value does this give the test? Should it be removed?

Reply:
It's useful for debugging, it was for me at least when writing the test. I think it's nice to have when you enable debug log.

Reply:
Can you elaborate? Why does logging the name help with debugging?

Reply:
No it was a copy paste from somewhere, I will just keep the else branch